lib/Service/RecordingService.php

```diff
@@ -142,4 +151,38 @@ private function getRecordingFolder(string $owner, string $token): Folder {
 		}
 		return $recordingFolder;
 	}
+
+	public function notifyStoredRecording(Room $room, Participant $participant, File $file): void {
+		$attendee = $participant->getAttendee();
+		$notificationLevel = $attendee->getNotificationLevel();
+		if ($notificationLevel === Participant::NOTIFY_NEVER) {
+			return;
+		}
+
+		$notification = $this->notificationManager->createNotification();
+
+		$notification
+			->setApp('spreed')
+			->setDateTime($this->timeFactory->getDateTime())
+			->setObject('chat', $room->getToken())
+			->setUser($attendee->getActorId())
+			->setSubject('record_file_stored', [
+				'objectType' => 'file',
+				'objectId' => $file->getId(),
+				'name' => $file->getName(),
+				'actorDisplayName' => $attendee->getDisplayName(),
+			]);
```

Comment:
It's quite clear that the object is a file on this subject?
With objectType and Id I meant the `$notification->setObject()` but maybe that's not a good idea.
So I would say let's remove `objectType`, `name` and `actorDisplayName` from the parameter list here as they are not needed.

```suggestion
			->setSubject('record_file_stored', [
				'objectId' => $file->getId(),
			]);
```